HotelReservation/src/main/java/HotelFinder.java

```diff
@@ -0,0 +1,47 @@
+import java.util.*;
+
+class SortArgument {
```

Comment:
Don't write multiple classes in a single file. Create different files for each class